### DISCUSSION TITLE
8.9.0 及以上版本 split 组件显示异常

### DISCUSSION THREAD

**Discussion by qinzouzou** [dotnetcore/BootstrapBlazor#4263] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

BootstrapBlazor 8.8.4及以下，split显示正常，升级到8.9.0及以上版本，重新生成解决方案后，显示异常。

![8 8 4](https://github.com/user-attachments/assets/42f9fa10-ddcc-48d7-8fec-f33161a40e14)
![8 9 0](https://github.com/user-attachments/assets/531e4dc8-477a-4993-a5cc-511c9a5fcd40)

### Steps To Reproduce

BootstrapBlazor 8.8.4及以下，split显示正常，升级到8.9.0及以上版本，重新生成解决方案后，显示异常。

[BootstrapBlazorApp1.zip](https://github.com/user-attachments/files/16940970/BootstrapBlazorApp1.zip)

### .NET Version

8.0.303

**Comment by bb-auto** [comment]
@qinzouzou Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply, marked as answer]
@qinzouzou 大概率是没有清缓存，我这里没问题，使用最新版即可
